internal/argparser/argparser.go

```diff
@@ -0,0 +1,78 @@
+package argparser
+
+import (
+	"errors"
+
+	"github.com/jsightapi/cli/internal/statistics"
+
+	"github.com/jsightapi/cli/internal/format"
+
+	"github.com/jsightapi/cli/internal/generator"
+
+	"github.com/urfave/cli/v2"
+
+	"os"
+)
+
+var noInputFileErrorString = "no input file"
+
+func DoWork(f format.Format, ctx *cli.Context) error {
```

Comment:
ctx is always better to pass as the first parameter

Reply:
fixed